tests/testthat/test_kernel.r

```diff
@@ -0,0 +1,6 @@
+context('kernel')
+
+status <- system2('python3', 'test_ir.py', wait=TRUE)
```

Comment:
It will be interesting what will happen if we submit that to CRAN: on my system, there is no python3 and in RStudio (or my r package install script), I don't have the kernel testing tool installed, so this tests will basically always fail.